packages/veritone-react-common/src/components/Breadcrumbs/index.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { shape, string, number, arrayOf, oneOfType, node, func } from 'prop-types';
+import { MenuList, Popover } from '@material-ui/core';
+import { ChevronRight, MoreVert } from '@material-ui/icons';
+import classNames from 'classnames';
+
+import BreadcrumbItem from './BreadcrumbItem';
+
+import styles from './Breadcrumbs.scss';
+
+export default class Breadcrumbs extends React.Component {
+  static propTypes = {
+    pathList: arrayOf(shape({
+      id: string.isRequired,
+    })),
+    maxItems: number,
+    seperator: oneOfType([string, node]),
+    onCrumbClick: func.isRequired
+  }
+
+  static defaultProps = {
+    maxItems: 5,
+    seperator: <ChevronRight className={styles['icon-color']} />,
+  }
+
+  state = {
+    openSpreadPath: false
+  }
+
+  onSpreadClick = (event) => {
+    this.anchorEl = event.target;
+    this.setState(({ openSpreadPath }) => ({
+      openSpreadPath: !openSpreadPath
+    }));
+  }
+
+  clickAway = () => {
+    this.anchorEl = null;
+    this.setState({
+      openSpreadPath: false
+    })
+  }
+
+  onCrumbClick = (event) => {
```

Comment:
Instead of using `data-*` attributes in the BreadcrumbItem, and spreading each crumb (`...crumb`) to pass data down, you can simply use a closure function to reference the data you need.

for example: 
`onCrumbClick = crumb => () => { ..... }`

then on each BreadcrumbItem you can just do this:
`onClick={this.onCrumbClick(crumb)}`